vagrant/kong/kuma/dataplane.yaml

```diff
@@ -0,0 +1,11 @@
+type: Dataplane
+mesh: default
+name: kong
+networking:
+  inbound:
```

Comment:
Demo should be updated to use the new `gateway` field that has been introduced specifically for this scenario